### ISSUE TITLE
@sanity/code-input 2.24.1 errors on setModuleUrl

### ISSUE THREAD

**Issue by gadkadosh** [sanity-io/sanity#3069]
**Describe the bug**

I get the following error with version 2.24.1 of code-input when `sanity graphql deploy`:

```
TypeError: Cannot read property 'setModuleUrl' of undefined
  at Object.<anonymous> (~/work/site/site/studio/node_modules/ace-builds/webpack-resolver.js:2:12)
  at Object.<anonymous> (~/work/site/site/studio/node_modules/@sanity/code-input/lib/editorSupport.js:5:1)
error Command failed with exit code 1.
```

**To Reproduce**

1. Update/install @sanity/code-input@2.24.1
2. Run sanity graphql deploy

**Which versions of Sanity are you using?**

Run `sanity versions` in the terminal and copy-paste the result here.

```
@sanity/cli                 2.23.2 (latest: 2.24.1)
@sanity/base                2.23.2 (latest: 2.24.1)
@sanity/cli                 2.23.2 (latest: 2.24.1)
@sanity/code-input          2.23.2 (latest: 2.24.1)
@sanity/color-input         2.23.2 (latest: 2.24.1)
@sanity/components          2.14.0 (up to date)
@sanity/core                2.23.2 (latest: 2.24.0)
@sanity/dashboard           2.23.2 (latest: 2.24.1)
@sanity/default-layout      2.23.2 (latest: 2.24.1)
@sanity/default-login       2.23.2 (latest: 2.24.1)
@sanity/desk-tool           2.23.2 (latest: 2.24.1)
@sanity/form-builder        2.24.1 (up to date)
@sanity/icons                  ??? (latest: 1.2.3)
@sanity/production-preview  2.15.0 (up to date)
@sanity/studio-hints        2.23.0 (latest: 2.24.1)
```

**What operating system are you using?**

MacOS / GitHub Actions (ubuntu)

**Which versions of Node.js / npm are you running?**

Run `npm -v && node -v` in the terminal and copy-paste the result here.
```
6.14.15
v14.18.3
```

**Comment by bjoerge** [comment]
Thanks for reporting! We're currently looking into this.

**Comment by bjoerge** [comment]
fixed in `@sanity/code-input@2.24.2`
